_posts/2015-03-16-aerogear-ios-2.2.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: AeroGear iOS SDK v2.2 is out !
```

Comment:
The post is missing author and categorization, see https://raw.githubusercontent.com/aerogear/aerogear.org/master/_posts/2015-01-30-aerogear-ios-2.1.md for inspiration.